.stylelintrc

```diff
@@ -4,6 +4,7 @@
     "no-duplicate-selectors": null,
     "font-family-no-duplicate-names": null,
     "no-descending-specificity": null,
-    "selector-type-no-unknown": null
+    "selector-type-no-unknown": null,
+    "at-rule-no-unknown": null
```

Comment:
Why was this added here rather than to the file or document that failed the lint rule?